### PR TITLE
feat: Parse defense in GuildAttackTimerOverlayFeature

### PR DESCRIPTION
**Pull request by kristofbolyai**
image

### CODE REVIEW

File: common/src/main/java/com/wynntils/core/objects/TimedSet.java [hunk 1]
```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright © Wynntils 2022.
+ * This file is released under AGPLv3. See LICENSE for full license details.
+ */
+package com.wynntils.core.objects;
```
Comment (Incompleteusern) [inline common/src/main/java/com/wynntils/core/objects/TimedSet.java]: Move to utils/objects, I'd rather we not make a new dir

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/objects/TimedSet.java]: It already existed

File: common/src/main/java/com/wynntils/wynn/model/GuildAttackTimerModel.java [hunk 2]
```diff
@@ -4,33 +4,84 @@
...
 
     private static List<TerritoryAttackTimer> attackTimers = List.of();
 
     public static void init() {}
 
+    @SubscribeEvent
+    public static void onMessage(ChatMessageReceivedEvent event) {
+        if (event.getRecipientType() != RecipientType.GUILD) return;
+
+        Matcher matcher = GUILD_DEFENSE_CHAT_PATTERN.matcher(event.getCodedMessage());
```
Comment (Incompleteusern) [inline common/src/main/java/com/wynntils/wynn/model/GuildAttackTimerModel.java:38]: Can be faked, but we can't do much about that.

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/wynn/model/GuildAttackTimerModel.java:38]: Yeah

File: common/src/main/java/com/wynntils/core/objects/TimedSet.java (hunk 1 above)
Comment (Incompleteusern) [reply common/src/main/java/com/wynntils/core/objects/TimedSet.java]: ?

Comment (Incompleteusern) [reply common/src/main/java/com/wynntils/core/objects/TimedSet.java]: This is a new file?

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/objects/TimedSet.java]: But the package existed

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/objects/TimedSet.java]: Moved it though

Comment (Incompleteusern) [reply common/src/main/java/com/wynntils/core/objects/TimedSet.java]: Odd, I think its only on your client

Comment (Incompleteusern) [reply common/src/main/java/com/wynntils/core/objects/TimedSet.java]: > Moved it though

Sorry to ask again, but this is not wynn specific and would be better off in the com.wynntils.utils.objects package

Comment (kristofbolyai) [reply common/src/main/java/com/wynntils/core/objects/TimedSet.java]: np moved

**Review by Incompleteusern — APPROVED** [review]

**Pull request merged**